### DISCUSSION TITLE
V14-RC2: Please review the UX of UUI-Modal implementation.

### DISCUSSION THREAD

**Discussion by markadrake** [umbraco/Umbraco-CMS#16173] [Features and ideas]
I work on a very large monitor, and I realize that not everybody does. This issue is obviously exaggerated by my circumstances, but please review the backoffice UX and, if possible, reduce the required amount of mouse movement.

![image](https://github.com/umbraco/Umbraco-CMS/assets/713355/d1560d31-f79e-4991-8ace-728ea3bc4fca)

![image](https://github.com/umbraco/Umbraco-CMS/assets/713355/c58ba4aa-347a-443f-9495-4eb95176930e)

![image](https://github.com/umbraco/Umbraco-CMS/assets/713355/7fa06dd3-190b-4c53-ae08-22b9a6594d4c)

![image](https://github.com/umbraco/Umbraco-CMS/assets/713355/4ad809fc-b649-435b-a59a-9f613cca2f59)

> Why did the "create modal" close after clicking "create folder"? IMO, this modal should remain visible. The "create folder" has no impact on the screen and does not present me with any new editing context. 

Now, repeat this multiple times throughout the application. It seems anything on the left-hand side that opens the UUI-Modal will do so on the right-hand side. It's only taken me 2 hours to tire of traversing from the left to the right side of the screen every time I want to perform any action.

**Comment by nielslyngsoe** [comment]
Hi @markadrake thanks for reporting this, we are fully aware that it is not optimal to have right aligned modals appearing from actions of the tree. Currently we are missing a proper technically way to left align modals. Yeah that sounds simple but with the stacking it requires a little more. When time allows that will be solved. But I can reassure you it is in our backlog.
Regarding the folder thats a good catch, I have copied it into an internal task of ours. :-)

With that I will close the discussion, let me know if you feel like I missed something :-)
